_observing-your-data/ad/index.md

```diff
@@ -102,6 +102,11 @@ To create a cross-cluster detector in OpenSearch Dashboards, the following [perm
       - `indices:data/read/search` -- The `search` permission is required in order to search custom results indexes to show results on the Anomaly Detection interface.
       - `indices:data/write/delete` -- The detector may generate many anomaly results. The `delete` permission is required in order to delete old data and save disk space.
       - `indices:data/write/bulk*` -- The `bulk*` permission is required because the plugin uses the Bulk API to write results into the custom index.
+   - There are a few nested fields in the custom result index mapping, making it challenging to aggregate and visualize the data effectively. To address this, we offer an option **Enable flattened custom result index** to flatten the custom result index.
+     - If you select this option for a custom results index, the plugin creates a separate index prefixed with the custom result index name and detector name. For example, if the detector `Test` uses the custom result index `abc`, a separate index with the alias `opensearch-ad-plugin-result-abc-flattened-test` will be created to store the anomaly detection results with nested fields flattened. In addition to creating a separate index, the plugin also sets up an ingest pipeline with a script processor. This pipeline is bound to the separate index and uses a Painless script to flatten all nested fields in the custom results index.
```

Comment:
can you add a link to your example and rules in result-mapping.md ?

Reply:
Will need to loop back around on the link. The build fails because the new section is currently unpublished.